packages/nimble-components/src/table/tests/table.spec.ts

```diff
@@ -136,10 +136,15 @@ describe('Table', () => {
         expect(document.createElement('nimble-table')).toBeInstanceOf(Table);
     });
 
+    it('element has a role of "grid"', async () => {
```

Comment:
similar to above can we exercise some table imperative creation pre-connect workflows form just calling setData in this file?